dsp-meta/resources/schema-metadata-draft.json

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
Is this draft something different than existing already `schema-metadata.json`?

Reply:
Yes, in our last meeting we decided to  mark all properties as optional which are not present for all json.